### PR TITLE
Fixes #47 multiple lines added in table generation

### LINKED ISSUE #47: Semantic  convention generator adds more than one blank line, failing mdlint
See https://github.com/open-telemetry/opentelemetry-specification/pull/1759#issuecomment-861645499

The PR in question has two enums in a single convention, and the first one has a note on one of the members. As a result, two consecutive newlines are printed between the footnote and the next enum table.

### PR DESCRIPTION
**Pull request by bryce-b**
also adds tests for said issue.

### CODE REVIEW

**Comment by bryce-b** [comment]
#47

**Review by thisthat — COMMENTED** [review]
Thank you :)

File: semantic-conventions/src/tests/data/markdown/multiple_enum/expected.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+| `unavailable` | unavailable |
+
+`net.host.connection.subtype` MUST be one of the following or, if none of the listed values apply, a custom value:
+
+| Value  | Description |
+|---|---|
+| `1G` | 1G
+ |
+| `2G` | 2G
+ |
```
Comment (thisthat) [inline semantic-conventions/src/tests/data/markdown/multiple_enum/expected.md:51]: I would expect:
```suggestion
| `2G` | 2G |
```

**Review by thisthat — APPROVED** [review]

**Review by mariojonke — APPROVED** [review]

**Review by Oberon00 — APPROVED** [review]
Existing tests were not affected? Great!

**Review by arminru — APPROVED** [review]

**Comment by arminru** [comment]
Thanks!

**Pull request merged**